sample-compose-fragments/src/main/java/uk/gov/hmrc/sample_compose_fragments/presentation/screens/ColorsListScreen.kt

```diff
@@ -0,0 +1,104 @@
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.dimensionResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import uk.gov.hmrc.components.compose.ui.theme.HmrcBlack
+import uk.gov.hmrc.components.compose.ui.theme.HmrcGrey3
+import uk.gov.hmrc.components.compose.ui.theme.HmrcWhite
+import uk.gov.hmrc.sample_compose_components.R
+import uk.gov.hmrc.sample_compose_fragments.domain.model.ColorItem
+import uk.gov.hmrc.sample_compose_fragments.presentation.fragment.ColorItemClickListener
+import uk.gov.hmrc.sample_compose_fragments.presentation.viewModel.ColorsViewModel
+
+@Composable
+fun ColorsListScreen(viewModel: ColorsViewModel, navigateTo: ColorItemClickListener) {
+    DisplayList(viewModel, navigateTo)
+}
+
+@Composable
+fun DisplayList(viewModel: ColorsViewModel, click: ColorItemClickListener) {
+    val listItems by viewModel.colorItems.collectAsState()
+    val listState = rememberLazyListState()
+
+    LazyColumn(
+        modifier = Modifier.fillMaxSize(1F).background(HmrcGrey3), state = listState
+    ) {
+        items(listItems) { item ->
+            ListItem(item = item, click)
+        }
+    }
+}
+
+@Composable
+fun ListItem(item: ColorItem, click: ColorItemClickListener) {
+    Card(
+        modifier = Modifier
+            .fillMaxWidth().padding(
+                dimensionResource(id = R.dimen.hmrc_spacing_16),
+                dimensionResource(id = R.dimen.hmrc_spacing_8)),
+        elevation = CardDefaults.cardElevation(defaultElevation = 1.dp),
+        colors = CardDefaults.cardColors(containerColor = HmrcWhite,), shape = RoundedCornerShape(0)
+    ) {
+        Row(
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier
+                .padding(dimensionResource(id = R.dimen.hmrc_spacing_12))
+                .clickable(onClick = { click(item) })
+        ) {
+            Box(
+                modifier = Modifier
+                    .heightIn(min = 40.dp).widthIn(min = 40.dp).border(BorderStroke(1.dp, HmrcBlack))
+                    .background(item.color).align(alignment = Alignment.CenterVertically),
+            )
+            Text(
+                modifier = Modifier
+                    .padding(horizontal = dimensionResource(id = R.dimen.hmrc_spacing_16)).align(alignment = Alignment.CenterVertically),
+                text = item.colorName, color = HmrcBlack, fontSize = 18.sp,textAlign = TextAlign.Center,
```

Comment:
Can we make the font sizes standard too please? Same as the padding sizes/colours.